source/predicates/array.ts

```diff
@@ -158,4 +160,25 @@ export class ArrayPredicate<T = unknown> extends Predicate<T[]> {
 			}
 		});
 	}
+
+	/**
+	Test all elements in the array to match the exact shape of the provided predicate.
+
+	The predicates is an array which describes how the tested array should look like. Each predicate placed at the nth index should be applied to the item at the same index.
+
+	@param predicates - The predicate placed at the nth index should be applied also to the item at the same index.
```

Comment:
This is just repeating the above. It should actually describe what `predicates` is.